src/app/components/ClientHomeInterface.tsx

```diff
@@ -15,10 +16,18 @@ const ClientHomeInterface = () => {
 
   const [selected, setSelected] = useState(defaultValue)
```

Comment:
Let's not use generic name for state, use name that defines/explains the actual functionality/usage